R/text_summary.R

```diff
@@ -27,11 +27,8 @@
 #' library(gsDesign)
 #'
 #' # Text summary of a 1-sided design
-#' x <- gs_design_ahr(info_frac = 1:3/3, test_lower = FALSE)
+#' x <- gs_design_ahr(info_frac = 1:3/3, test_lower = FALSE) %>% to_integer()
 #' x %>% text_summary()
-#' x %>% to_integer() %>% text_summary()
-#' gs_power_ahr(event = c(10, 20, 30), test_lower = FALSE) %>% text_summary()
-#' gs_power_ahr(event = c(10, 20, 30), test_lower = FALSE) %>% to_integer() %>% text_summary()
```

Comment:
If these examples would be informative for end users, you can wrap them in `\donttest{}` like I recently did for {simtrial} in https://github.com/Merck/simtrial/pull/332. It's your decision. Though if you do decide to use `\donttest{}`, you have to remember to re-document the package to update the `.Rd` file

Reply:
Thanks for reminding me of the `\donottest{}`. After further consideration, I have decided to remove these examples, as they primarily demonstrate the implementation of `gs_power_ahr`, whether it is information-based or event-based. They do not effectively emphasize the use case of `text_summary`.